repository/src/main/resources/alfresco/repository.properties

```diff
@@ -1,1247 +1,1285 @@
+
+# Maximum query size for category/tag fetch when not explicitly set by paging parameters
+category.queryFetchSize=5000
+
+# Brute force protection
+authentication.protection.enabled=true
+authentication.protection.limit=10
+authentication.protection.periodSeconds=6
+
+system.email.sender.default=noreply@alfresco.com
+# reset password workflow will expire in an hour
+system.reset-password.endTimer=PT1H
+system.reset-password.sendEmailAsynchronously=true
+
+# HeartBeat 
+heartbeat.target.url=
+heartbeat.enabled=true
+
+# CSRF filter overrides
+csrf.filter.enabled=true
+csrf.filter.referer=
+csrf.filter.referer.always=false
+csrf.filter.origin=
+csrf.filter.origin.always=false
+
+# CORS settings
+cors.enabled=false
+cors.allowed.origins=
+cors.allowed.methods=GET,POST,HEAD,OPTIONS,PUT,DELETE
+cors.allowed.headers=Authorization,Content-Type,Cache-Control,X-Requested-With,accept,Origin,Access-Control-Request-Method,Access-Control-Request-Headers,X-CSRF-Token
+cors.exposed.headers=Access-Control-Allow-Origin,Access-Control-Allow-Credentials
+cors.support.credentials=true
+cors.preflight.maxage=10
+
+# Alfresco Rest Api-Explorer
+api-explorer.url=
+
+# Events subsystem
+events.subsystem.autoStart=false
+# Messaging subsystem
+messaging.subsystem.autoStart=true
+
+
+# Raw events
+acs.repo.rendition.events.endpoint=jms:acs-repo-rendition-events?jmsMessageType=Text
+
+# Transform request events
+acs.repo.transform.request.endpoint=jms:acs-repo-transform-request?jmsMessageType=Text
+
+# If enabled doesn't allow to set content properties via NodeService
+contentPropertyRestrictions.enabled=true
+contentPropertyRestrictions.whitelist=
+
+# Repo events2
+# Type and aspect filters which should be excluded
+# Note: System folders node types are added by default
+repo.event2.filter.nodeTypes=sys:*, fm:*, cm:thumbnail, cm:failedThumbnail, cm:rating, rma:rmsite include_subtypes
+repo.event2.filter.nodeAspects=sys:*
+repo.event2.filter.childAssocTypes=rn:rendition
+# Comma separated list of users which should be excluded
+# Note: username's case-sensitivity depends on the {user.name.caseSensitive} setting
+repo.event2.filter.users=
+# Topic name
+repo.event2.topic.endpoint=amqp:topic:alfresco.repo.event2
+# Thread pool for async enqueue of repo events
+repo.event2.queue.enqueueThreadPool.priority=1
+repo.event2.queue.enqueueThreadPool.coreSize=8
+repo.event2.queue.enqueueThreadPool.maximumSize=10
+# Thread pool for async dequeue and delivery of repo events
+repo.event2.queue.dequeueThreadPool.priority=1
+repo.event2.queue.dequeueThreadPool.coreSize=1
+repo.event2.queue.dequeueThreadPool.maximumSize=1
+
+
+# MNT-21083
+# --DELETE_NOT_EXISTS - default settings
+system.delete_not_exists.batchsize=100000
+system.delete_not_exists.delete_batchsize=1000
+system.delete_not_exists.read_only=false
+system.delete_not_exists.timeout_seconds=-1
+system.prop_table_cleaner.algorithm=V2
+
+# Configure the system-wide (ACS) settings for direct access urls.
+#
```

Comment:
I think it is worth adding a comment about the different layers and the precedence between them.

Also a line about having different content stores and connectors and how different ones should have names that follow a defined pattern.

Reply:
done (taken from design spec)